lib/trader.js

```diff
@@ -63,9 +63,18 @@ Trader.prototype.syncLogs = function syncLogs () {
   this.request({ path: '/logs', method: 'get' })
     .then(data => data.body)
   // Delete log files that are two or more days old
-    .then(() => {
-      let today = new Date()
-      return logs.removeLogFiles(today.setDate(today.getDate() - 2))
+    .then((it) => {
+      const twoDaysAgo = (() => {
+        let date = new Date()
+
+	// Notice that `setDate()` can take negative values. So if you'd take
+	// -2 days on the 1st of April you'd get the 30th of March. Several
+	// examples can be seen at: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate#Using_setDate()
+        date.setDate(date.getDate() - 2)
+	return date
+      })()
+
+      return logs.removeLogFiles(twoDaysAgo).then(it)
```

Comment:
`then()` expects a function as argument. This should be `then(() => it)`

Reply:
Thanks for the review, Rafi!

Reply:
It was mainly regarding the bug, though. Didn't look deep into the PR.